.github/scripts/process_commit.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+"""
+This script finds the merger responsible for labeling a PR by a commit SHA. It is used by the workflow in
+'.github/workflows/pr-labels.yml'. If there exists no PR associated with the commit or the PR is properly labeled,
+this script is a no-op.
+
+Note: we ping the merger only, not the reviewers, as the reviewers can sometimes be external to pytorch
+with no labeling responsibility, so we don't want to bother them.
+This script is based on: https://github.com/pytorch/vision/blob/main/.github/process_commit.py
+"""
+
+import sys
+from typing import Any, Set, Tuple
+
+import requests
+
+# For a PR to be properly labeled it should have release notes label and one topic label
+PRIMARY_LEBEL_FILTER = "release notes:"
+SECONDARY_LEBEL_FILTER = "topic:"
```

Comment:
I think it actually might be better to explicitly list the topics here since we don't generally expect the topics to change (they are consistent across all categories).

Reply:
@anjali411 for consistency I would rather use same approach for release notes and topics. As you mentioned before, its easier to curate the release notes items and topics simply by adding them to available labes.
However with your new suggestion, we want "release notes" to be retrieved from github available labels and topics hardcoded ?

Reply:
yeah I think it might be better to hardcode topics because we don't usually want to add new topics and we shouldn't allow people to add new topics without a consensus

Reply:
Ok Sounds good. Let me do it the way you suggested.